torch/overrides.py

```diff
@@ -1012,7 +1012,60 @@ def get_testing_overrides() -> Dict[Callable, Callable]:
         torch.vstack: lambda tensors, out=None: -1,
         torch.where: lambda condition, x=None, y=None: -1,
         torch.zeros_like: lambda input, dtype=None, layout=None, device=None, requires_grad=False: -1,
+        torch._make_dual_copy: lambda primal, tangent, level: -1,
```

Comment:
Looks like I needed these to appease __torch_function__ tests. I noticed that not all of the existing view ops appear in this list though, which seems a little weird (`Tensor.view`?)

Reply:
The list here looks off; we shouldn't be adding these as methods. Was the method addition unintentional? 
view is on the list: Tensor.view: lambda self, shape: -1,

Reply:
Right now in the PR, I look at whether a given `{view}` op has a method or function variant (or both) and generate the same for the `{view}_copy` variant. I guess since this is all sugar that isn't really intended to be used directly in python this doesn't matter too much. Would you rather everything just be a function and not a method?

Reply:
yes, functions only please

Reply:
done